src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectDetailsAjaxController.java

```diff
@@ -119,4 +123,62 @@ public ProjectDetailsAjaxController(ProjectService projectService, UIProjectsSer
 					.body(new AjaxErrorResponse(e.getMessage()));
 		}
 	}
+
+	/**
+	 * Update the priority for analyses for a project.
+	 *
+	 * @param projectId identifier for a {@link Project}
+	 * @param priority  the new priority for analyses
+	 * @param locale    current users locale
+	 * @return message to user about the update ot the priority
+	 */
+	@PutMapping("/priority")
+	public ResponseEntity<AjaxResponse> updateProcessingPriority(@RequestParam long projectId,
+			@RequestParam AnalysisSubmission.Priority priority, Locale locale) {
+		try {
+			return ResponseEntity.ok(
+					new AjaxSuccessResponse(service.updateProcessingPriority(projectId, priority, locale)));
+		} catch (UpdateException e) {
+			return ResponseEntity.status(HttpStatus.BAD_REQUEST)
+					.body(new AjaxErrorResponse(e.getMessage()));
+		}
+	}
+
+	/**
+	 * Update the minimum/maximum coverage or genome size for the project
+	 *
+	 * @param projectId identifier for the project
+	 * @param coverage  minimum/maximum coverage or genome size for the project
+	 * @param locale    current users locale
+	 * @return Message to user about the update
+	 */
+	@PutMapping("/coverage")
+	public ResponseEntity<AjaxResponse> updateProcessingCoverage(@RequestParam long projectId,
+			@RequestBody Coverage coverage, Locale locale) {
+		try {
+			return ResponseEntity.ok(
+					new AjaxSuccessResponse(service.updateProcessingCoverage(coverage, projectId, locale)));
+		} catch (UpdateException e) {
+			return ResponseEntity.status(HttpStatus.BAD_REQUEST)
+					.body(new AjaxErrorResponse(e.getMessage()));
+		}
+	}
+
+	/**
+	 * Delete a project
+	 *
+	 * @param projectId identifier for a project
+	 * @return an indication to the user about the result of the update
+	 */
+	@PostMapping("/delete")
```

Comment:
Can we just use the `DeleteMapping` here?

Reply:
Smart man, fixed in 97c16ce